packages/graph/hash_graph/lib/graph/src/store/postgres/knowledge/entity/mod.rs

```diff
@@ -553,7 +553,7 @@ impl<C: AsClient> EntityStore for PostgresStore<C> {
                     .attach_printable("cannot update link order of an entity that is not a link")
             ),
             (Some(link_data), left, right) => {
-                let new_left_order = match (link_data.left_order(), left) {
+                let new_left_order = match (link_data.left_to_right_order(), left) {
```

Comment:
You missed this line 🙂

Reply:
Cheers, that made me think of a few more things to find and replace